### ISSUE TITLE
Bump to latest versions of PINRemoteCache and PINRemoteImage

### ISSUE THREAD

**Issue by mc7h** [TextureGroup/Texture#1713]
"Beta 7" of PINCache has a serious bug(s) that were causing disk caches be deleted upon instantiation.

This bug has been fixed in "Beta 8". We use PINCache directly in our app (as well as transitively through Texture) the bug - not present in "Beta 6" - has prevented us from moving to the latest versions of Texture.

**Comment by ay8s** [comment]
It looks like the Podspec does pull in PINRemoteImage 3.0.0-beta.14 which includes 3.0.1-beta.8 of PINCache. 

Can you check if something else is keeping PINCache at the older version? Or try using master for Texture to see if that bumps things up? 2.8.1 however should work.

**Comment by mc7h** [comment]
> PINRemoteImage 3.0.0-beta.14 which includes 3.0.1-beta.8 of PINCache.

No, PINRemoteImage 3.0.0-beta.14 uses "3.0.1-beta.7" in both the podspec and the Cartfile.

So does the Texture Cartfile.

**Comment by mc7h** [comment]
https://github.com/pinterest/PINRemoteImage/releases/tag/3.0.0

3.0.0 has now been released; bumping to this version will also pull in 3.0.1-beta.8.
